harness/nans.js

```diff
@@ -1,9 +1,12 @@
-/**
- * A collection of NaN values produced from expressions that have been observed
- * to create distinct bit representations on various platforms. These provide a
- * weak basis for assertions regarding the consistent canonicalization of NaN
- * values in Array buffers.
- */
+// Copyright (C) 2017 Ecma International.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: |
```

Comment:
For continuous text, you can use `>`. AFAIK, the pipe preserves the formatting with the link breaks.

This can be done in a follow up PR